2018/10/16/src/main/java/me/lemire/microbenchmarks/algorithms/Utf8Validate.java

```diff
@@ -227,12 +229,12 @@ public boolean testJava_utf8(MyState state) {
 
 
 
-  public static void main(String[] args) throws RunnerException {
-        Options opt = new OptionsBuilder()
-        .include(Utf8Validate.class.getSimpleName()).warmupIterations(3)
-        .measurementIterations(3).forks(1).build();
-        new Runner(opt).run();
-
-  }
+//  public static void main(String[] args) throws RunnerException {
```

Comment:
Remove the main here so the main in JMH can do it's work. I guess in principle you can leave them both in, so you can select either on the command line? Maybe better?